crates/wasm/test/shared.ts

```diff
@@ -0,0 +1,31 @@
+import { initialiseResolver } from "@noir-lang/noir-source-resolver";
+import { compile } from "../dist/";
```

Comment:
Note that we need to place the downloaded wasm package artifact back into `./crates/wasm` so that it can resolve the `@noir-lang/noir-source-resolver` dependency.